README.md

```diff
@@ -137,481 +137,4 @@ SELECT * FROM MT_DEVICES will show the current vouchers created by DI messages.
-| DELETE /api/v1/owner/vouchers/?id=<device_guid> | Deletes voucher of the specified GUID from the `TO2_DEVICES` table. | Query - id: Device GUID | | | |
-| POST /api/v1/owner/svivalues/?id=\<serviceinfo_id>&isCborEncoded=<boolean_value> | Adds serviceinfo entry to `OWNER_SERVICEINFO` table. The query parameter 'isCborEncoded' should be 'true' for CBOR encoded binary data that will never be split into smaller chunks while transferring to the device (small in length, to be used for CBOR primitives such as boolean, int, array and map), and it should be 'false' (preferably) for other binary data that could be splitted into smaller chunks and transferred across messages (for example binary values, string). | Query - id: Service info ID, isCborEncoded: Boolean | application/octet-stream or application/cbor | Content of Serviceinfo in binary format. | |
-| DELETE /api/v1/owner/svivalues/?id=<serviceinfo_id> | Deletes the serviceinfo values from `OWNER_SERVICEINFO` table. | Query - id: Service info ID | | | |
-| POST /api/v1/owner/svi/?guid=<device_guid> | Adds owner serviceinfo for the GUID in `GUID_OWNERSVI` table, that will be transferred to the device in the insertion order. The format is `Entry1,Entry2,Entry3` and so on, where each Entry contains `moduleName:messageName=serviceInfoId`. Here, the 'content' corresponding to the 'serviceInfoId' is transferred to the device. Please see \<fido-iot-src\>/demo/owner/serviceinfo/sample-svi.csv as an example for the above format, where moduleName is 'sdo_sys' and messageName is either one of 'filedesc', 'write' and 'exec'. The order of each 'Entry' is important as this order decides the sequence in which the Owner will transfer the Service info. | Query - guid: Device GUID | application/text | SVI string | |
-| DELETE /api/v1/owner/svi/?guid=<device_guid> | Deletes owner serviceinfo for the GUID from the `GUID_OWNERSVI` table. | Query - guid: Device GUID | | | |
-
-***NOTE*** These REST APIs use Digest authentication. `owner_api_user` and `owner_api_password` properties specify the credentials to be used while making the REST calls.
-
-### Configuring the FIDO IoT HTTP Java Device Sample
-
-Some software settings are runtime-configurable via Java properties.  They include:
-
-- `fido.iot.randoms`
-
-  A comma-separated list of Java `SecureRandom` algorithm names for random number generation.
-  Values are in order from least to most preferred.
-
-  Default is `"NativePRNG,Windows-PRNG"`.
-
-- `fido.iot.url.di`
-
-  The URL at which the Device Initialization (DI) server may be found.
-
-  Default is `http://localhost:8039/`.
-
-- `fido.iot.pem.dev`
-
-  The location of the PEM file containing the device keys (private and public).
-  If not set, a hardcoded key is used - see the Java source for details.
-
-  There is no default. Provide value './device.pem' to use the existing default EC-256 key-pair.
```

Comment:
Please add this line into device's README.

Reply:
Done